dbms/tests/integration/image/Dockerfile

```diff
@@ -21,8 +22,11 @@ RUN apt-get update && env DEBIAN_FRONTEND=noninteractive apt-get install --yes -
     libicu-dev \
     bsdutils \
```

Comment:
libicu & bsdutils are clickhouse dependencies, not sure if they are needed in that container

Reply:
added here: https://github.com/yandex/ClickHouse/commit/0901694b1560f779b1623df481a24309fc200091